src/main/java/net/shadowmage/ancientwarfare/npc/entity/NpcCombat.java

```diff
@@ -201,7 +201,7 @@ public void writeEntityToNBT(NBTTagCompound tag) {
 	@Override
 	public void attackEntityWithRangedAttack(EntityLivingBase target, float force) {
 		// minimum inaccuracy = 3.0f, slowly reaches 0 (or close to it) as the NPC reaches max level
-		float inaccuracy = 3.0f - ((float) Math.sqrt(getLevelingStats().getBaseLevel()) / (float) Math.sqrt(AWNPCStatics.maxNpcLevel) * 5.0f);
+		float inaccuracy = 3.0f - ((float) Math.sqrt(getLevelingStats().getBaseLevel()) / (float) Math.sqrt(AWNPCStatics.maxNpcLevel) * 3.0f);

```

Comment:
@ThatPolishKid99 why this change? I believe we discussed this previously and even though it's confusing it worked correctly, or is that not the case?

Reply:
This prevents inaccuracy going into negative numbers so it's alright to merge